docs/sources/Date-and-time-values.md

```diff
@@ -72,6 +72,24 @@ occurred during that specific hour.
 * [File Times](https://docs.microsoft.com/en-us/windows/win32/sysinfo/file-times)
 * [Precision and accuracy of DateTime](https://docs.microsoft.com/en-us/archive/blogs/ericlippert/precision-and-accuracy-of-datetime), by Eric Lippert, April 8, 2010
 
+## .NET DateTime
+### Characteristics
+
+Attribute | Description
+--- | ---
+Supported date range | ...
```

Comment:
From https://docs.microsoft.com/en-us/dotnet/api/system.datetime?view=net-6.0

The DateTime value type represents dates and times with values ranging from 00:00:00 (midnight), January 1, 0001 Anno Domini (Common Era) through 11:59:59 P.M., December 31, 9999 A.D. (C.E.) in the Gregorian calendar.